catalog/transformer/catalog-transformer-pptx/pom.xml

```diff
@@ -166,6 +166,7 @@
                             org.slf4j.*,
                             org.w3c.dom.*,
                             org.xml.sax.*,
+                            org.codice.ddf.platform.util
```

Comment:
:question: how come this import package statement was needed here but nowhere else?

Reply:
Because of the weird way this bundle is made.